metrics-service-ui/data-source-plugin/src/QueryEditor.tsx

```diff
@@ -2,47 +2,68 @@ import defaults from 'lodash/defaults';
 
 import React, { ChangeEvent, PureComponent } from 'react';
 import { LegacyForms } from '@grafana/ui';
-import { QueryEditorProps } from '@grafana/data';
+import { QueryEditorProps, SelectableValue } from '@grafana/data';
 import { DataSource } from './datasource';
 import { defaultQuery, MyDataSourceOptions, MyQuery } from './types';
 
 const { FormField } = LegacyForms;
+const availableClustersEndpoint = 'https://localhost:10010/metrics-service/api/v1/clusters';
```

Comment:
Can the host and port be parameterized? E.g. environment variables when the plugin is run?